app/controllers/events_controller.rb

```diff
@@ -0,0 +1,61 @@
+class EventsController < ApplicationController
+  def index
+    @events = []
+    @event = Event.all
+    @setting = Setting.all.first.holidays
+    @events << @event
+    @events << @setting
+  end
+
+  def new
+    @event = Event.unscoped.new
+  end
+
+  def home
+    @events = Event.all
+  end
+
+  def create
+    @event = Event.unscoped.new(event_params)
```

Comment:
`

current_company.events or
current_employee.company.events
`

Reply:
why was this comment not resolved? reason?